libfuse/fs.go

```diff
@@ -354,6 +354,12 @@ func (f *FS) Statfs(ctx context.Context, req *fuse.StatfsRequest, resp *fuse.Sta
 		Namelen: ^uint32(0),
 		Frsize:  fuseBlockSize,
 	}
+	if _, err := libkbfs.GetCurrentSessionIfPossible(
+		ctx, f.config.KBPKI(), false); err != nil {
```

Comment:
Did you test this?  I think it's the wrong usage of `GetCurrentSessionIfPossible`.  It returns a nil error if you're logged in, but with a zero-value session.  That way you can still return unexpected errors.